"main" key in package.json is invalid path

```
> require('bluebird-events');
Uncaught:
Error: Cannot find module '/dist/index.js'. Please verify that the package.json has a valid "main" entry
```

this is caused due to package.json listing
```
  "main": "/dist/index.js",
  "types": "/dist/index.d.ts",
```
those paths should instead be relative, ie "./dist/index.js" (the periods are missing)

--- a/package.json
+++ b/package.json
@@ -2,8 +2,8 @@
   "name": "bluebird-events",
   "version": "3.0.1",
   "description": "A wrapper around event emitters that returns a bluerbird promise resolved or rejected when certain events are fired.",
-  "main": "/dist/index.js",
-  "types": "/dist/index.d.ts",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
   "engines": {
     "node": ">=8.0.0"
   },

